### PR TITLE
More sfx

### PR DESCRIPTION
**Pull request by cmeese549**
Featuring:
1. ships crashing
2. ships exploding
3. hit marker
4. kill confirmation
5. player get shot
6. targeting sound

### CODE REVIEW

File: Assets/Scripts/Audio/SFX/SFXController.cs [hunk 1]
```diff
@@ -65,7 +64,7 @@ public void PlayRocketLaunchCooldownSFX (AudioSource source)
         source.Stop();
         source.clip = missileSounds[0];
         source.Play();
-        StartCoroutine(PlayAfterDelay(source, combatSounds[1], 1f, .3f));
+        StartCoroutine(PlayAfterDelay(source, combatSounds[1], 1f, .1f));
```
Comment (jakemcleman) [inline Assets/Scripts/Audio/SFX/SFXController.cs]: I know this is old code but I didn't notice before. I'm not a fan of always using a predefined index here, if combatSounds has a bunch of sounds that should be used at different times shouldn't they all be named for those different times?

File: Assets/Scripts/Audio/SFX/SFXController.cs [hunk 2]
```diff
@@ -81,6 +80,51 @@ public void PlayUpgradedSound (AudioSource source)
...
+        //so I check if it's null
+        //this is probably bad
+        if (source == null) return;
+        int rand = Random.Range(0, crashSounds.Length);
+        source.PlayOneShot(crashSounds[rand], 4f);
+    }
+
+    public void PlayHitmarkerSound (AudioSource source)
+    {   
+        source.PlayOneShot(combatSounds[2], 2f);
```
Comment (jakemcleman) [inline Assets/Scripts/Audio/SFX/SFXController.cs]: Please make hit marker sounds and kill confirmed sound their own variables, relying on remembering and getting the correct indices seems gross and error prone

File: Assets/Scripts/EnemyAI/EnemyHealth.cs [hunk 3]
```diff
@@ -16,6 +16,12 @@ public class EnemyHealth : MonoBehaviour, IHealth
 
     private bool hasSpawned;
 
+    //If you are seeing this I forgot to go look for an enemy type enum
+    public string enemyType;
```
Comment (jakemcleman) [inline Assets/Scripts/EnemyAI/EnemyHealth.cs]: You said to block on making this an enum so this is me doing that

**Review by jakemcleman — CHANGES_REQUESTED** [review]

File: Assets/Scripts/Audio/SFX/SFXController.cs (hunk 1 above)
Comment (cmeese549) [reply Assets/Scripts/Audio/SFX/SFXController.cs]: lol fuckin ludum jank, idk how I saw that system yesterday and just chose to ignore it

File: Assets/Scripts/EnemyAI/EnemyHealth.cs (hunk 3 above)
Comment (cmeese549) [reply Assets/Scripts/EnemyAI/EnemyHealth.cs]: https://github.com/jacobmcleman/LD46/pull/30/commits/053c13cddaed007ba5d55564b142b2383f1bf926

File: Assets/Scripts/Audio/SFX/SFXController.cs (hunk 2 above)
Comment (cmeese549) [reply Assets/Scripts/Audio/SFX/SFXController.cs]: https://github.com/jacobmcleman/LD46/pull/30/commits/fb0ede07ee286d0a37be569db8cbbebc6fc351c9

File: Assets/Scripts/Audio/SFX/SFXController.cs (hunk 1 above)
Comment (cmeese549) [reply Assets/Scripts/Audio/SFX/SFXController.cs]: https://github.com/jacobmcleman/LD46/pull/30/commits/fb0ede07ee286d0a37be569db8cbbebc6fc351c9

**Comment by cmeese549** [comment]
Changes made and things tested, I'm ready to merge this now if you are

**Review by jakemcleman — APPROVED** [review]
lgtm

**Pull request merged**